Gemfile

```diff
@@ -31,6 +31,9 @@ group :development, :test do
 
   # Have jQuery local for testing
   gem 'jquery-rails'
+
+  # Needed for deploying to dev/test
+  gem 'coffee-rails'
```

Comment:
Ahhhh, that makes sense why it wasn't needed before.  I redact my criticism of rails.  Rails is Life. Rails is Love.